src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -1213,7 +1213,9 @@ private function registerTranslatorConfiguration(array $config, ContainerBuilder
             $container->getDefinition('console.command.translation_update')->replaceArgument(6, $transPaths);
         }
 
-        if ($container->fileExists($defaultDir)) {
+        if (null == $defaultDir) {
```

Comment:
```suggestion
        if (null === $defaultDir) {
```

Reply:
Oops, sorry. Thank you. It looks like I write in too many programming languages. I will commit your suggestion as soon as the way to go is decided.

Reply:
I had to fix this once again, once I knew it by heart, but I miss some training ...